### DISCUSSION TITLE
What is this Error? “Some repos have failed to update” ? How to fix it?

### DISCUSSION THREAD

**Discussion by TheNonSmoker** [Fox2Code/FoxMagiskModuleManager#270] [Q&A] [answered]
![Screenshot_20221228-154004_Fox's Magisk Module Manager](https://user-images.githubusercontent.com/30073064/209796045-e12cbc9c-a31e-4ab3-bb7e-9ea5cbf15e90.png)

**Comment by androidacy-user** [comment, marked as answer]
Exactly what it says. One or more repos have failed to update. Such issues are usually temporary or due to network issues.

In the future, please use Discussions or Telegram for support requests.

**Reply by TheNonSmoker** [reply]
Yes exactly, I also later got it as a network issue, but this only happens when I'm on my mobile network, app shows updates fine when I connect to a Wi-Fi, I know my mobile internet is fine and other apps also runs fine, but only this app comes up with this issue only with my mobile internet.
Can you please guide me, how can I fix this? Settings seems all ok for me, this app has the permission to use my mobile sim card internet in the settings.

**Reply by androidacy-user** [reply]
Unfortunately, there's not much we can do: it's a problem with your mobile network. You might try using a VPN or different network.

**Reply by TheNonSmoker** [reply]
It's ok, thanks for the assistance, I think it's an issue with my custom ROM, I hope I find a solution soon!
